uploadComplete.php

```diff
@@ -996,13 +996,13 @@
 
 <table class="headerTable">
 <tr><td>
-<div class="headerText">Status</div>
+<div class="headerText"><?php echo _("Status");?></div>
 	<br />
-    <p>File archived as <?php echo $Base_URL . 'archive/' . $uploadedFilename; ?>.</p>
-    <p>Log file available at: <a href='<?php echo $Base_URL . $logfile; ?>'><?php echo $Base_URL . $excelfile; ?></a>.</p>
-    <p>Process completed.  <?php echo $mailOutput; ?></p>
+    <p><?php echo _("File archived as").' '.$Base_URL . $uploadedFile; ?>.</p>
+    <p><?php echo _("Log file available at:");?> <a href='<?php echo $Base_URL . $logfile; ?>'><?php echo $Base_URL . $excelfile; ?></a>.</p>
+    <p><?php echo _("Process completed.")."  ".$mailOutput; ?></p>
     <br />
-    Summary:<?php echo $rownumber . " titles processed.<br />" . nl2br($logSummary); ?><br />
+    <?php echo _("Summary:").'  '.$rownumber . _(" titles processed.")."<br />" . nl2br($logSummary); ?><br />

```

Comment:
Does Gettext provide a way to include variables in a translation?  Or maybe we should just use PHP's sprintf().  In this code example, I think we need something like _("Summary: %d titles processed."), where the "%d" allows translators to decide where the number should appear in the translation.